src/pages/blog/automate-where-it-matters-with-process-maps.pug

```diff
@@ -0,0 +1,143 @@
+
+            p In conclusion, instead of simply increasing leads, we'd be much better off looking at why so many customers drop out from the website or resort to using the phone. We could then work to make it easier and more convenient for customers to carry on online.
+
+            h3 A helpful workflow
+
+            p As we work to add automation to our client's businesses <a href="/" target="_blank" rel="noopener">at Leaf</a>, we use process maps to help both ourselves and stakeholders achieve a shared understanding of their businesses, identify where improvements can be made, and where money is best invested to make the biggest positive impact. We follow this workflow:
+
+            ol
+              li <strong>Build a process map</strong> - Sometimes this will be for the entire business, and sometimes a particular process within the business. As a general guideline, we aim for a map detailed enough that it accurately reflects 80% of the possible scenarios in the process we're trying to improve.
+              li <strong>Identify areas for improvement</strong> - With a map in place, we can begin to identify the main areas for improvement. Typically these are things that are time consuming or expensive to complete, cause a significant dropoff, or otherwise introduce some sort of bottleneck to efficiency.
+              li <strong>Create goals</strong> - Once we know where we want to improve, we can create S.M.A.R.T goals around these areas. Side note - we wrote <a href="https://weareleaf.com/blog/goals-matter" target="_blank" rel="noopener">a whole post on how to use goals effectively</a>.
+              li <strong>Assign goals</strong> - At this point, we've got a list of goals we need to achieve in order to make the business process better. We assign each to a capable person or team to complete.
+              li <strong>Build and work backlogs</strong> - Teams and individuals build backlogs of work focussed on achieving their goal, and work through them.
+              li <strong>Repeat</strong> - Once the goals have been achieved (or failed to be achieved), we update our process map to reflect the impact of the changes and repeat the process.
+
+            p With this workflow, we're systematically rooting out inefficiencies, figuring out where we can have the biggest positive impact, using our knowledge of the numbers to set realistic goals, and building up focussed roadmaps around those goals to give stakeholders confidence in the value being delivered.
+
+            p This is effective because work is driven from a solid <strong>shared understanding</strong> of the business, and a <strong>shared agreement</strong> about how where best to focus energy on improving things. If necessary, we can course correct after each iteration of the workflow. This ensures a continued focus and improvement in the right areas.
```

Comment:
'Course correct' doesn't feel popular enough to me to be confident of it being easily understood by all. I'd go with 'correct our course' or 'alter our plan of action' or similar.